doc/faq.rst

```diff
@@ -24,7 +24,69 @@ There are a couple of reasons for holding off on this for now:
-mechanism at some point. Stay tuned.
+mechanism at some point.
+
+The `NEXTLEAP EU project <https://nextleap.eu>`_ has done some R&D
+regarding key verification questions and protection against active
+network-adversary attacks, see `the countermitm verification protocols
+<https://countermitm.readthedocs.io/en/latest/new.html>`_ some of which
+are implemented with the Autocrypt-enabled `DeltaChat <https://delta.chat>`_ app.
+DeltaChat implemented "setup contact" and "verified group join" work flows
+which try to tackle key verification without talking to users about keys.
+This has not yet been fully discussed within the Autocrypt context but
+it's an interesting approach to look at as it is implemented already
+and received a first round of thorough user-testing late 2018.
+
+How does Autocrypt differ from PEP?
+---------------------------------------------------------------
+
+Autocrypt and PEP are different approaches to opportunistic e-mail encryption.
+
+The PEP effort predates Autocrypt by a few years and involves a foundation
+and two companies which provide FOSS and paid offerings.  PEP offers the
+GPL-licensed PEP-Engine and adapters which can be integrated into MUAs
+if licensing permits.  Some specification efforts have started but
+as far as we know there are no independent implementations
+of the PEP protocols. The PEP organizations maintain own
+`software products that integrate the pEp engine <https://pep.software/>`_.
+
+Autocrypt was created in December 2016 by several independent mail app implementors
+and folks from the e2e e-mail encryption space. Afterwards, they publically
+collaborated, conducted specification sessions, early user-testings and learnt from
+each others early implementations efforts. At the end of 2017 they settled and published
+the Autocrypt Level 1 specification which was updated to version 1.1 in February 2019.
+The specification is implemented by a growing number of independent mail
+apps, allowing for cross-app end-to-end email encryption. Note that
+Autocrypt does not offer any software downloads but supports MUA implementors
+to add Autocrypt support. Autocrypt has has no formal organization, company or
+funding as of February 2019. Autocrypt is jointly run and cared for by independent
+actors who typically aim at reaching consensus among each other,
+and who are well-known in the OpenPGP and e-mail encryption community
+of which they are a part of.
+
+There are also a few technical differences despite both efforts supporting
+opportunistic key management, for example:
+
+- PEP generates one key per device and encrypts to multiple keys for one
+  recipient. Autocrypt only uses one key per e-mail address, to reduce
+  security concerns and implementation complexity.
+
+- Autocrypt MUAs specifies how to share the encryption setup between different
+  mail apps and this is implemented and reported to work between
+  different Autocrypt-enabled MUAs.  PEP does not offer sharing of the
```

Comment:
Could it be that "Autocrypt MUAs specifies" should actually be "Autocrypt specifies", and "MUAs " only slipped in by accident?

Also this sentence could be split up, leaving out the "and". Easier to understand.